CourseGrab/ViewControllers/Home/HomeTableViewCell.swift

```diff
@@ -183,6 +183,12 @@ class HomeTableViewCell: UITableViewCell {
             }
         }
     }
+    
+    override var isHighlighted: Bool {
+        didSet{
+            containerView.backgroundColor =  isHighlighted ? UIColor(red: 220 / 255, green: 220 / 255, blue: 220 / 255, alpha: 1) : .white
```

Comment:
You have a double space 

```suggestion
            containerView.backgroundColor = isHighlighted ? UIColor(red: 220 / 255, green: 220 / 255, blue: 220 / 255, alpha: 1) : .white
```